test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -167,9 +164,9 @@ define([
         });
 
         const messagesToSend = [
-            [jcm.REQUESTS.STATUS, { [JOB_CHANNEL]: 0 }],
-            [jcm.REQUESTS.STATUS, { [JOB_CHANNEL]: 1 }],
-            [jcm.REQUESTS.STATUS, { [JOB_CHANNEL]: 2 }],
+            [jcm.REQUESTS.STATUS, { [JOB_ID]: 0 }],
+            [jcm.REQUESTS.STATUS, { [JOB_ID]: 1 }],
+            [jcm.REQUESTS.STATUS, { [JOB_ID]: 2 }],
```

Comment:
This should have been the `JOB_ID` param, rather than referring to the job channel.